packages/wallets/src/components/WalletsAccordion/WalletsAccordion.scss

```diff
@@ -0,0 +1,24 @@
+.wallets-accordion {
+    width: 100%;
+    background: var(--system-light-8-primary-background, #fff);
+    border-radius: 16px;
+
+    &__header {
+        background: none;
+        border: none;
+        color: inherit;
+        outline: inherit;
+        cursor: pointer;
+        width: 100%;
+        display: flex;
+        align-items: center;
+        padding: 24px;
+        gap: 24px;
+    }
+
+    &__dropdown {
+        &--open {
+            transform: rotate(180deg);
```

Comment:
@farhan-nurzi-deriv please add the transition with duration for smooth expand/collapse